src/components/Heading/enums.ts

```diff
@@ -0,0 +1,3 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+export type HeadingLevel = '1' | '2' | '3' | '4' | '5' | '6';
+export type HeadingSize = HeadingLevel | 'none';
```

Comment:
@unindented do you think these options make sense?

Reply:
They made sense to me.

Reply:
I guess why `none` instead of `null | undefined`?

Reply:
default `undefined` value would use the current HeadingLevel style, so the unstyled text option needs an actual value. i think it's fine but it's a little odd